src/raft.rs

```diff
@@ -1447,6 +1458,10 @@ impl<T: Storage> Raft<T> {
                         self.election_elapsed = 0;
                         self.vote = m.from;
                     }
+                    // This means it's in split vote, give up election.
+                    if self.judge_split_prevote && self.state == StateRole::PreCandidate {
```

Comment:
It seems if this peer grants its (pre)vote to another one, it should reset `election_elapsed` or become follower.

Reply:
This can lead to an awkward situation that A and B all grant prevotes to each other and all become follower.

Reply:
Got it.